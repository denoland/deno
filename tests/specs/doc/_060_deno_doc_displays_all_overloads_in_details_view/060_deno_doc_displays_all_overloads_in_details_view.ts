// deno-lint-ignore-file
export namespace NS {
  export function test(name: string, fn: Function): void;
  export function test(options: object): void;
  export function test(name: string | object, fn?: Function): void {}
}
