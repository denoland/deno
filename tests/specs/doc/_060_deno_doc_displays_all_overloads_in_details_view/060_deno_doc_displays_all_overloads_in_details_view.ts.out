Defined in [WILDCARD]/060_deno_doc_displays_all_overloads_in_details_view.ts:3:3

function test(name: string, fn: Function): void

Defined in [WILDCARD]/060_deno_doc_displays_all_overloads_in_details_view.ts:4:3

function test(options: object): void


